### PR TITLE
add new post

### PR DESCRIPTION
**Pull request by Any97Cris**

### CODE REVIEW

File: source/_posts/electronic-signatures.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Free and open source software for electronic signatures
 author: Daiane Alves
 date: 2024-01-04
 description: Agility and security are paramount in business transactions, and LibreSign emerges as the intelligent choice for diverse sectors. Developed by LibreCode, a cooperative of IT professionals, LibreSign embodies the Free and Open-Source Software (FOSS) philosophy. With robust security standards, it ensures the inviolability of electronic signatures, making it ideal for government, education, and corporate enterprises
-categories: [features]
+categories: ''
```
Comment (vitormattos) [inline source/_posts/electronic-signatures.md]: Is this necessary?
What's the impact when the category is empty?
At home we filter the post with category features, not?!

Comment (Any97Cris) [reply source/_posts/electronic-signatures.md]: If this option does not have break the code.

Comment (vitormattos) [reply source/_posts/electronic-signatures.md]: Here:
https://github.com/LibreSign/site/blob/4e5ad9dd54ab6d599e6923edfdf5fb7e128cda9f/source/index.blade.php#L59
We filter the posts by category.
When we set the category to empty, don't will return any post at home.
And categories need to be an array, but isn't mandatory.
Categories are used to classify posts and apply filters.

File: config.php [hunk 2]
```diff
@@ -124,7 +124,7 @@
                     }
                 }
             }
-            if (is_array($post['categories']) && in_array($category, $post['categories'])) {
+            if (isset($post['categories'])) {
```
Comment (vitormattos) [inline config.php]: in_array is important to return only the defined category.
Without this will return all posts that have category.

**Pull request merged**